src/NaNMath.jl

```diff
@@ -354,4 +354,164 @@ for f in (:min, :max)
     @eval ($f)(a, b, c, xs...) = Base.afoldl($f, ($f)(($f)(a, b), c), xs...)
 end
 
+"""
+    NaNMath.findmin([f,] domain) -> (f(x), index)
+
+##### Args:
+* `f`: a function applied to the values in `domain`
+* `domain`: A non-empty iterable of floating point numbers or `Missing`.
+
+##### Returns:
+* Returns a pair of a value in the codomain (outputs of `f`, defaulting to `identity`) and
```

Comment:
The default value of `f` should be mentioned in the description of the argument, not in the "Returns" section